### PR TITLE
Upgrade React to 15.x

### PR DESCRIPTION
**Pull request by vgrbr**
* Upgraded React to 15.x, added `prop-types` & `create-react-class` npm packages.
* Added used babel presets
* Upgraded browserify, babelify to latest major versions

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-svg-icons",
-  "version": "0.2.0",
+  "version": "0.4.0",
```
Comment (felixhageloh) [inline package.json]: all looks good. Just one question: why go from 0.2 to 0.4 directly :)?

File: index.js [hunk 2]
```diff
@@ -1,17 +1,19 @@
 var React = require('react');
+var createReactClass = require('create-react-class');
+var PropTypes = require('prop-types');
 var toReact = require('svg-to-react');
 var assign = require('object-assign');
 var resolveAssetPath = require('./lib/resolve-asset-path');
 
 var icons = {};
-var Icon = React.createClass({
+var Icon = createReactClass({
```
Comment (felixhageloh) [inline index.js]: is it possible to make this a functional component?

**Review by suweller — APPROVED** [review]

File: index.js [hunk 3]
```diff
@@ -1,49 +1,48 @@
...
+}
 
-  getDefaultProps: function() {
-    return {
-      color: function (_, original) {
-        return original;
-      }
-    };
-  },
+var Icon = function Icon(props) {
```
Comment (felixhageloh) [inline index.js]: un-necessary `var` :)

Comment (vgrbr) [reply index.js]: @felixhageloh  fixed :)

**Review by felixhageloh — APPROVED** [review]

**Pull request merged**